### DISCUSSION TITLE
Scraping an SPA

### DISCUSSION THREAD

**Discussion by internadine** [FlowiseAI/Flowise#1156] [Q&A]
Hi everyone, I want to scrape my own SPA to provide a consulting Chatbot. I have tried the Puppeteer-node, but it does not work. It does scrape other pages without an issue. Any ideas? I tried different settings for additional parameters, but so far no luck :-( Thanks, Nads

**Comment by mikehudson2** [comment]
Try using the Apify node

**Reply by balvirsingh** [reply]
is it free?
